x/market/keeper/deal_test.go

```diff
@@ -516,5 +552,29 @@ func makeTestCert() types.DataValidationCertificate {
 		UnsignedCert: &uCert,
 		Signature:    sign,
 	}
+}
+
+func makeTestDataCert2() types.DataValidationCertificate {
+	uCert := types.UnsignedDataValidationCertificate{
+		DealId:               2,
+		DataHash:             []byte("1a312c1223x2fs3"),
+		EncryptedDataUrl:     []byte("https://panacea.org/a/123.json"),
+		DataValidatorAddress: newAddr.String(),
+		RequesterAddress:     acc2.String(),
```

Comment:
It seems to me that only this address is a different value.
How about it takes an address as a parameter?

Reply:
Ah you means the function likes ```func makeTestDataCert(address sdk.AccAddress) { ... }``` ?

Reply:
That's right.

Reply:
Not only different for address, but also for ```dataHash```. So I will takes a ```dataHash``` and ```address``` as a parameter.

Reply:
Thank you :)